http2-tests/src/test/scala/org/apache/pekko/http/impl/engine/http2/HttpMessageRenderingSpec.scala

```diff
@@ -13,14 +13,14 @@
 
 package org.apache.pekko.http.impl.engine.http2
 
+import com.typesafe.config.ConfigFactory
 import java.time.format.DateTimeFormatter
 import org.apache.pekko
+import org.apache.pekko.http.scaladsl.settings.{ ClientConnectionSettings, ServerSettings }
```

Comment:
Can you fix this import so it matches the same style of imports currently in the file